src/openfermion/chem/molecular_data.py

```diff
@@ -403,7 +406,8 @@ def spinorb_from_spatial(one_body_integrals, two_body_integrals):
 
 
 class MolecularData(object):
-    """Class for storing molecule data from a fixed basis set at a fixed
+    """Class for storing molecule data from a fixed basis set at a fixed.
```

Comment:
This doesn't look like it's a correct period, since it is continued "fixed ....  period"

Reply:
You're right. That was an auto-formatting result. The tool must have gotten confused by the blank line after the first line. Will fix.